plasmapy/dispersion/dispersionfunction.py

```diff
@@ -90,11 +128,38 @@ def plasma_dispersion_func(
     if not np.all(np.isfinite(zeta)):
         raise ValueError("The argument to plasma_dispersion_function is not finite.")
 
-    Z = 1j * np.sqrt(np.pi) * Faddeeva_function(zeta)
+    return plasma_dispersion_func_lite(zeta)
 
-    return Z
 
+@preserve_signature
+def plasma_dispersion_func_deriv_lite(zeta: numbers.Real) -> numbers.Real:
 
+    r"""
```

Comment:
```suggestion
def plasma_dispersion_func_deriv_lite(zeta):
    r"""
```

For the same reason as `plasma_dispersion_func`.